src/content/developers/docs/apis/json-rpc/index.md

```diff
@@ -5,19 +5,19 @@ lang: en
 sidebar: true
 ---
 
-[JSON](http://json.org/) is a lightweight data-interchange format. It can represent numbers, strings, ordered sequences of values, and collections of name/value pairs.
+A remote procedure call, or RPC, is a method of retrieving or being notified of information from a given data source in an asynchronous manner. [JSON-RPC](http://www.jsonrpc.org/specification) is a stateless, light-weight RPC protocol, that defines several data structures and the rules around their processing. It is transport agnostic in that the concepts can be used within the same process, over sockets, over HTTP, or in many various message passing environments. It uses [JSON](http://json.org/) as data format.
```

Comment:
I might be wrong here but this feels like a generic explanation of what an RPC is? Is there something we can add to provide context of why it's relevant to Ethereum development? Maybe that's obvious from this overview if you're a developer but it's not clear to me.

Reply:
Happy to... Was on the fence about keeping it more generic, but can add some tie-in here.